### PR TITLE
R4R: Mark decimal is required

### PR DESCRIPTION
**Pull request by zhangyelong**
Resolves: #1813

### CODE REVIEW

File: docs/cli-client/asset/issue-token.md [hunk 1]
```diff
@@ -20,10 +20,10 @@ iriscli asset issue-token [flags]
 | --gateway          | string  | false    |               | The unique moniker of the gateway, required when the source is gateway |
 | --symbol           | string  | true     |               | The symbol of the token, length between 3 and 6, alphanumeric characters, case insensitive |
```
Comment (haifengxi) [inline docs/cli-client/asset/issue-token.md]: `Symbol` length should be between 3 and 8.

Please double check all the related docs to make sure their contents match our spec and implementation, and that English-Chinese versions are consistent.

File: docs/zh/cli-client/asset/issue-token.md [hunk 2]
```diff
@@ -20,10 +20,10 @@ iriscli asset issue-token [flags]
 | --gateway          | string  | false    |               | 网关的唯一标识, 当 source 为 gateway 时必填                  |
 | --symbol           | string  | true     |               | Source内全局唯一的资产标识符，字母和数字的组合，首字符必须为字母，长度[3,8]，大小写无关 |
 | --canonical-symbol | string  | false    |               | 当 source 为 gateway 时，用来指定源链上的 Symbol，字母和数字的组合，长度[3,8]，大小写无关 |
```
Comment (haifengxi) [inline docs/zh/cli-client/asset/issue-token.md]: We don't enforce validation rule on `CanonicalSymbol` any more.

File: docs/cli-client/asset/issue-token.md (hunk 1 above)
Comment (zhangyelong) [reply docs/cli-client/asset/issue-token.md]: Done, double checked

File: docs/zh/cli-client/asset/issue-token.md (hunk 2 above)
Comment (zhangyelong) [reply docs/zh/cli-client/asset/issue-token.md]: Done

**Pull request merged**